<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenswanAuth</class>
 <widget class="QWidget" name="OpenswanAuth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>136</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="groupNameLabel">
     <property name="text">
      <string>Group name:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>leGroupName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="leGroupName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="userPasswordLabel">
     <property name="text">
      <string>User password:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>leUserPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="leUserPassword">
     <property name="passwordMode">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="groupPasswordLabel">
     <property name="text">
      <string>Group password:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>leGroupPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KLineEdit" name="leGroupPassword">
     <property name="passwordMode">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cbShowPasswords">
     <property name="text">
      <string>Show passwords</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
